### ISSUE TITLE
Sonarr v3 issue

### ISSUE THREAD

**Issue by mhertel** [trossr32/sonarr-radarr-lidarr-autosearch-browser-extension#21]
I am running both radarr and sonarr v3. I have changed the settings in advanced as instructed.
Radarr works just fine, but in Sonarr I just get an empty search box.

for example clicking on the icon in IMDb for the series 24, it sends me to sonarr with this url:
http://192.168.1.10:8989/add/new/24
but the search box is empty..

Version 3.0.4.994
Package Version 3.0.4.994-ls71 by linuxserver.io
Mono Version 5.20.1.34

**Comment by trossr32** [comment]
Apologies I've only just seen this, will try to replicate and provide a solution.

**Comment by mhertel** [comment]
No worries, we're all busy with other stuff these days :)

**Comment by trossr32** [comment]
This appears to be an issue with the extension code that populates the search field and implements the search running slightly before the search field is rendered. I have added an element watch function wait until the input is available before performing the tasks, so hopefully this will fix the issue going forward.

Implemented as v1.3.2 and submitted to all web stores, so will be updated once the package has been through review on each browser's store.
